pkg/bgpv1/test/fixtures.go

```diff
@@ -218,6 +218,8 @@ func setup(ctx context.Context, peerConfigs []gobgpConfig, fixConfig fixtureConf
 			peer.stopGoBGP()
 		}
 
+		f.bgp.BGPMgr.Stop()
```

Comment:
Can't we hook this `Stop()` into the Hive's Stop lifecycle callback?

Reply:
Including Stop in hive would mean that in prod code hive would call gobgp STOP while shutting down. This terminates all neighbors by sending PEER_DECONFIGURED, which results in peer not going into helper mode of graceful restart. 

So, calling this in test cleanup code - so we free up the BGP listen port.